Entity/Entity/Services/VirtualShop.cs

```diff
@@ -0,0 +1,169 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Entity.Services
+{
+    public class VirtualShop
+    {
+        public List<IProductable> MyProducts { get; set; }
+        public List<ICompanable> MyCompanies { get; set; }
+        public List<ICategoriesable> MyCategories { get; set; }
+        public List<IProductsFromCompanable> MyProductsFromCompanies { get; set; }
+
+        public VirtualShop()
+        {
+            MyProducts = new List<IProductable>();
+            MyCompanies = new List<ICompanable>();
+            MyCategories = new List<ICategoriesable>();
+            MyProductsFromCompanies = new List<IProductsFromCompanable>();
+            StartValue();
+        }
+        private void StartValue()
+        {
+            MyCategories.Add(new Categories { Id = 1, Name = "Cakes" });
+            MyCategories.Add(new Categories { Id = 2, Name = "Pie" });
+
+            MyCompanies.Add(new Company { Id = 1, Name = "Fast Cat" });
+            MyCompanies.Add(new Company { Id = 2, Name = "Gold Fish" });
+            MyCompanies.Add(new Company { Id = 3, Name = "Big Sun" });
+
+            MyProducts.Add(new Product { Id = 1, Name = "The Best Cakes", CategoryId = 1 });
+            MyProducts.Add(new Product { Id = 2, Name = "Same Cake", CategoryId = 1 });
+            MyProducts.Add(new Product { Id = 3, Name = "Orang Pie", CategoryId = 2 });
+            MyProducts.Add(new Product { Id = 4, Name = "Aple Pie", CategoryId = 2 });
+            MyProducts.Add(new Product { Id = 5, Name = "Streng Cakes", CategoryId = 1 });
+            MyProducts.Add(new Product { Id = 6, Name = "Special Cakes", CategoryId = 1 });
+            MyProducts.Add(new Product { Id = 7, Name = "Syper Cakes", CategoryId = 1 });
+
+            MyProductsFromCompanies.Add(new ProductsFromCompany { CompanyId = 1, ProductId = 1 });
+            MyProductsFromCompanies.Add(new ProductsFromCompany { CompanyId = 1, ProductId = 2 });
+            MyProductsFromCompanies.Add(new ProductsFromCompany { CompanyId = 1, ProductId = 3 });
+            MyProductsFromCompanies.Add(new ProductsFromCompany { CompanyId = 1, ProductId = 4 });
+            MyProductsFromCompanies.Add(new ProductsFromCompany { CompanyId = 1, ProductId = 5 });
+            MyProductsFromCompanies.Add(new ProductsFromCompany { CompanyId = 2, ProductId = 1 });
+            MyProductsFromCompanies.Add(new ProductsFromCompany { CompanyId = 2, ProductId = 2 });
+            MyProductsFromCompanies.Add(new ProductsFromCompany { CompanyId = 2, ProductId = 6 });
+            MyProductsFromCompanies.Add(new ProductsFromCompany { CompanyId = 3, ProductId = 7 });
+        }
+        public void GetAllProduct()
+        {
+            Speaker.Output("===========================================================================");
+            Speaker.Output("All Product");
+
+            //var products = from e in MyProduct
```

Comment:
yeah this is on of the solution
but either uncomment it and move to separate method or remove this commented code